plasma_framework/contracts/src/utils/FailFastReentrancyGuard.sol

```diff
@@ -1,21 +1,21 @@
 pragma solidity 0.5.11;
 
+import "../framework/PlasmaFramework.sol";
+
 /**
```

Comment:
Might need to revisit this documentation + consider rename (probably don't need) ?

Reply:
done